From b94f36534bbeb7eb1af2ea99fd2c5b43978184dc Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Fri, 24 Apr 2015 17:11:45 +0100
Subject: [PATCH] Add Makefile

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 Makefile | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..bdb217b
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,16 @@
+datadir ?= /usr/share
+SHELL = /bin/bash
+
+dummy:
+
+install-data:
+	mkdir -p $(DESTDIR)$(datadir)/icons/ ; \
+	for iconset in "elementary-xfce" "elementary-xfce-dark" "elementary-xfce-darker" "elementary-xfce-darkest" ; do \
+	    cp -Rv $$iconset/ $(DESTDIR)$(datadir)/icons/. ; \
+	    gtk-update-icon-cache -f $(DESTDIR)$(datadir)/icons/$$iconset ; \
+	done; \
+	chmod a+r -R $(DESTDIR)$(datadir) ;
+
+install: install-data
+
+.PHONY: dummy
-- 
2.7.0

